static/js/message_events.js

```diff
@@ -377,7 +377,14 @@ export function update_messages(events) {
                 post_edit_topic = pre_edit_topic;
             }
             const args = [event.stream_id, pre_edit_topic, post_edit_topic, new_stream_id];
-            recent_senders.process_topic_edit(...args);
+            recent_senders.process_topic_edit({
```

Comment:
We need to do this for all the `event_messages`.

Reply:
Fix coming.

Reply:
Fixed.